### PR TITLE
windows: updated docs to include info regarding https validation

### PR DESCRIPTION
**Pull request by jborean93**
##### SUMMARY
Updated Windows docs to contain more information regarding the latest pywinrm release.

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
windows docs

##### ANSIBLE VERSION
```
2.5
```

### CODE REVIEW

File: docs/docsite/rst/windows_winrm.rst [hunk 1]
```diff
@@ -657,6 +666,34 @@ for each authentication option. See the section on authentication above for more
...
+When using a self-signed certificate or setting
+``ansible_winrm_server_cert_validation: ignore`` these security mechanisms are
+bypassed. While self signed certificates will always need the ``ignore`` flag,
+certificates that have been issued from a certificate authority can still be
+validated.
+
+One of the more common ways of setting up a HTTPS listener in a domain
+environment is to use Active Directory Certificate Service (AD CS). AD CS is
+used by Active Directory to sign certificate requests that are sent to it and
+then issue a certificate from the csr. The issuer certificate of the AD CS
```
Comment (dharmabumstead) [inline docs/docsite/rst/windows_winrm.rst]: Explain what a “CSR” is. In general, define abbreviations on first use and then you can just use the abbreviation (capitalized, please) for subsequent mentions.

File: docs/docsite/rst/windows_winrm.rst [hunk 2]
```diff
@@ -657,6 +666,34 @@ for each authentication option. See the section on authentication above for more
...
+instance can then be exported as a PEM encoded certificate. This file can then
+be copied locally to the Ansible controller.
+
+This file can either contain the single issuer certificate or multiple
+certificates. To then use this file as part of the valiation process, set
+``ansible_winrm_ca_trust_path`` to the path of the. If this value is not set
```
Comment (dharmabumstead) [inline docs/docsite/rst/windows_winrm.rst]: Please either remove the word “which” or add parenthesis around that part of the sentence so it makes it easier to understand.

File: docs/docsite/rst/windows_winrm.rst [hunk 3]
```diff
@@ -657,6 +666,34 @@ for each authentication option. See the section on authentication above for more
...
+then the certificate chain which is located in the ``cacert.pem`` installed by
+the `certifi <https://github.com/certifi/python-certifi>`_ package is used.
+
+.. Note:: Each HTTP call is done by the Python requests library which does not
+    use the systems built in certificate store as a trust authority.
```
Comment (dharmabumstead) [inline docs/docsite/rst/windows_winrm.rst]: “system’s built-in”

**Review by dharmabumstead — CHANGES_REQUESTED** [review]
A few edit requests. Thanks @jborean93!

**Comment by jborean93** [comment]
Thanks @dharmabumstead, I've made a few edits around some of the sentences so it is ready for review again.

**Pull request merged**